src/core/distributions/phylogenetics/tree/birthdeath/StateDependentSpeciationExtinctionProcess.cpp

```diff
@@ -585,7 +585,12 @@ double StateDependentSpeciationExtinctionProcess::computeRootLikelihood( void )
     {
         node_likelihood[i] = left_likelihoods[i];
 
-        if ( use_cladogenetic_events == true && speciation_node == true )
+        // MRM 03/23/2020: the root is a cladogenetic event, so the
+        // cladogenetic events should be included at the root. right now,
+        // I am just forcing the cladogenetic events at the root, but there
+        // may be a better solution.
+        if ( use_cladogenetic_events == true || speciation_node == true )
```

Comment:
I'm probably wrong here, but maybe you would want `if (use_cladogenetic_events && (speciation_node or root_node))`.